### PR TITLE
Checkout UI: handling when the user has no account

### PR DESCRIPTION
**Pull request by cellog**
# Description

Introducing the "you got no wallet" modal.

Note this adds a new color that is unused anywhere else, so I decided to name it "off-black" but @smombartz you're welcome to weigh in on the correct name for the color

Screen Shot 2019-06-05 at 6 20 30 PM
Screen Shot 2019-06-05 at 6 18 20 PM
Screen Shot 2019-06-05 at 6 30 36 PM
Screen Shot 2019-06-05 at 6 30 30 PM

# Issues

Refs #2903 

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: paywall/src/components/checkout/NoWallet.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+export const NoWallet = ({ config }: Props) => {
+  return (
+    <React.Fragment>
+      <Header>
+        <Title>
+          {config.icon && <Logo src={config.icon} />}
+          <UnlockedText>Unlocked</UnlockedText>
+        </Title>
+        <p>
+          To enjoy Forbes Online without any ads you&apos;ll need to use a
```
Comment (cellog) [inline paywall/src/components/checkout/NoWallet.tsx]: Note: this text will need to become part of the paywall configuration as a follow-up

**Comment by julien51** [comment]
Let's make sure @smombartz looks at this please before merging.

**Comment by cnasc** [comment]
> Note this adds a new color that is unused anywhere else, so I decided to name it "off-black" but @smombartz you're welcome to weigh in on the correct name for the color

image

It looks like `--offblack` is lighter than `--darkgrey`, so it should probably take a name somewhere in the grey hierarchy

LG in general

**Comment by smombartz** [comment]
the 7D grey should be in use for Labels, here are the colors and names i've been using

image

**Comment by cellog** [comment]
Fixes #3457

**Comment by cellog** [comment]
we use this currently:

```
    --brand: #ff6771;
    --white: #ffffff;
    --red: #ed663a;
    --lightred: #ffb79f;
    --offwhite: #f6f6f6;
    --lightgrey: #eeeeee;
    --grey: #a6a6a6;
    --dimgrey: #6a6a6a;
    --silver: #d8d8d8;
    --darkgrey: #4a4a4a;
    --slate: #333333;
    --link: #4d8be8;
    --green: #74ce63;
    --activegreen: #59c245;
    --darkgreen: #368043;
    --pink: #ed6e82;
    --yellow: #f6c61b;
```

So can we find a name that is consistent without needing to change the whole app?

**Comment by smombartz** [comment]
mediumgrey?

**Comment by cellog** [comment]
closing, there are some changes to make

**Pull request closed without merging**